src/common/options.cc

```diff
@@ -5850,6 +5850,14 @@ std::vector<Option> get_mds_client_options() {
     .set_default(true)
     .set_description(""),
 
+    Option("min_caps_per_client", Option::TYPE_INT, Option::LEVEL_ADVANCED)
+    .set_default(100)
+    .set_description(""),
```

Comment:
New config options must have a description and docs added in `doc/cephfs/client-config-ref.rst`.

Reply:
@batrick 
added it

Reply:
@renhwztetecs The description in the above code needs updated.

Reply:
@batrick updated it

Reply:
@batrick updated it